babel.config.js

```diff
@@ -53,8 +45,7 @@ module.exports = {
   presets: defaultPresets.concat(['@babel/preset-react']),
   plugins: [
     'babel-plugin-optimize-clsx',
-    ['@babel/plugin-proposal-class-properties', { loose: true }],
```

Comment:
Using strict accounts for the regular bundle size increase. Since the plan is to rely on vendor implementations in the future we should never use loose mode for production code.

Reply:
I agree in theory but I disagree in practice. I don't that it matters in our case. Class properties are mostly internal concern, well tested.

Reply:
Looking back at the past 20 years of javascript I think it's safe to say that loose mode is never a good idea.

Reply:
It remembers me when I first used [6to5](https://babeljs.io/blog/2015/01/12/6to5-esnext/). Upgrading a version would break multiple of the class syntax wrote.